src/interfaces/ICrossChainRelayerPayable.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.16;
+
+import "../libraries/CallLib.sol";
+
+/**
+ * @title CrossChainRelayer interface
+ * @notice CrossChainRelayer interface of the ERC-5164 standard as defined in the EIP.
+ */
+interface ICrossChainRelayerPayable {
+  /**
+   * @notice Custom error emitted if the `gasLimit` passed to `relayCalls`
+   *         is greater than the one provided for free on the receiving chain.
+   * @param gasLimit Gas limit passed to `relayCalls`
+   * @param maxGasLimit Gas limit provided for free on the receiving chain
+   */
+  error GasLimitTooHigh(uint256 gasLimit, uint256 maxGasLimit);
```

Comment:
I guess this is cut now?  No max?

Reply:
Yes, removed in this PR: https://github.com/pooltogether/ERC5164/pull/9